pgserver/connection_handler.go

```diff
@@ -1425,21 +1430,20 @@ func (h *ConnectionHandler) handleCopyToStdout(query ConvertedQuery, copyTo *tre
 					cancel()
 					return
 				}
-				copyData := &pgproto3.CopyData{
-					Data: line,
+				if flag {
+					flag = false
+					count := bytes.Count(line, []byte{'\t'})
+					sendCopyOutResponse(count + 1)
 				}
-				logrus.Warnf("line: %s", line)
-				sendCopyData(copyData)
+				sendCopyData(line)
```

Comment:
It would be better to let sendCopyData return error and check the returned error here to terminate the loop if the send call failed.